srcpkgs/rtaudio/template

```diff
@@ -11,9 +11,12 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="http://www.music.mcgill.ca/~gary/rtaudio/"
 distfiles="http://www.music.mcgill.ca/~gary/rtaudio/release/rtaudio-${version}.tar.gz"
-checksum=407680389ab4ef1270f9b2f6b953f063e27319dbe914de4394b6c3a1233d8f1a
+checksum=ff138b2b6ed2b700b04b406be718df213052d4c952190280cf4e2fab4b61fe09
```

Comment:
Heh, they make a new tarball with same name with an empty directory `.deps` removed and remove `x` permission ;)

Reply:
Yep. Sometimes upstream changes it randomly and that’s not a good thing